### PR TITLE
Pseudocode for longestPalindromicSubsequence using longestCommonSubse…

### PR DESCRIPTION
**Pull request by yildirimyigit**
This time, the code is very short. But I'm not sure how to refer to the lcs method that we assume we already have.

### CODE REVIEW

**Comment by yildirimyigit** [comment]
CPP code is given here https://gist.github.com/yildirimyigit/cafbfa233c72dc75445c54fcb63a44a1


**Note** : The code used for validation is not the same as the pseudocode.

**Comment by ghost** [comment]
Just copy paste my code (above it)

File: Pseudocode/Dynamic Programming/Longest Palindromic Subsequence (using LCS)/SourceCode.tex [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+
+  \caption{Find the length of the longest palindromic subsequence}
+
+  \begin{algorithmic}[1]
+    \Require A non empty string
+    \Statex
+
+    \Function{$Longest\_Palindromic\_Subsequence$}{$str$}
+        \Statex
+        \State $r\_str \gets reverse(str) j$
```
Comment (ghost) [inline Pseudocode/Dynamic Programming/Longest Palindromic Subsequence (using LCS)/SourceCode.tex]: Typo : Remove `j`

Comment (ghost) [reply Pseudocode/Dynamic Programming/Longest Palindromic Subsequence (using LCS)/SourceCode.tex]: Maybe you can write `rev_str` instead of `r_str`

**Review by ghost — CHANGES_REQUESTED** [review]

File: Pseudocode/Dynamic Programming/Longest Palindromic Subsequence (using LCS)/SourceCode.tex [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+        \State \Return{$longest\_common\_substring(str, r\_str)$}
```
Comment (ghost) [inline Pseudocode/Dynamic Programming/Longest Palindromic Subsequence (using LCS)/SourceCode.tex]: Horrible Bug here. I think you've mixed up the concepts. It should be `subsequence`. (If that was a typo, it's fine, just correct it).

**Review by ghost — CHANGES_REQUESTED** [review]

File: Pseudocode/Dynamic Programming/Longest Palindromic Subsequence (using LCS)/SourceCode.tex (hunk 2 above)
Comment (yildirimyigit) [reply Pseudocode/Dynamic Programming/Longest Palindromic Subsequence (using LCS)/SourceCode.tex]: Yes, of course. My bad again, sorry.

**Comment by yildirimyigit** [comment]
Ok. All should be fine now

File: Pseudocode/Dynamic Programming/Longest Palindromic Subsequence (using LCS)/SourceCode.tex [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+\DeclareCaptionLabelFormat{algnonumber}{Algorithm}
+\captionsetup[algorithm]{labelformat = algnonumber}
+
+\begin{document}
+
+\begin{algorithm}
+
+  \caption{Find the length of the longest palindromic subsequence}
+
+  \begin{algorithmic}[1]
```
Comment (ghost) [inline Pseudocode/Dynamic Programming/Longest Palindromic Subsequence (using LCS)/SourceCode.tex:26]: Also copy the `Ensure` Line from my previous code

**Review by ghost — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
Great. Merged!

**Comment by ghost** [comment]
I just moved the file into a different folder. Please sync your fork accordingly.

**Comment by yildirimyigit** [comment]
It was fun. I'm taking a break now but I may come back later.

Thanks @Just-A-Visitor for all your help

**Pull request merged**